sharktank/sharktank/models/punet/tools/import_brevitas_dataset.py

```diff
@@ -93,6 +88,19 @@ def _get_json_tensor(
         shape = qp[f"{name}_shape"]
         return torch.tensor(data_1d, dtype=dtype).reshape(shape)
 
+    if layer_name.endswith("output_softmax_quant"):
```

Comment:
A little bit of explanation here so that future us who rewrites this can figure it out?

Reply:
Added.